dockerfile/anaconda-release/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile to enable releases of Anaconda for RHEL independently of the host system.
+#
+# This will be based on our existing anaconda-rpm container because it has all the dependencies
+# required to build Anaconda.
+
+FROM quay.io/rhinstaller/anaconda-rpm:rhel-8
+LABEL maintainer=anaconda-list@redhat.com
+
+# Add missing dependencies required to do the build.
+RUN set -e; \
+  dnf install -y \
+  git \
+  python3-pip; \
+  dnf clean all;
+
+RUN pip3 install \
+  polib \
+  pocketlint
```

Comment:
I think some of the other containers run `pip3 install --upgrade pip (..)` - is that not needed on RHEL? That would be  good, of course, just not sure.

Reply:
I would prefer to update and upgrade all packages.

Reply:
That's directly at odds with the idea that the Python packages will be stable, packaged by the distro, as opposed to whatever is new on pip. The `pip --upgrade pip` thing was a hotfix, IIRC.